extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -647,6 +667,23 @@ public Builder setSpecialInstructions(final Object specialInstructions) {
             return this;
         }
 
+        /**
+         * Set whether to generate _metadata and _common_metadata files while writing parquet files.
+         * <ul>
+         * <li>When writing a single parquet file, metadata files are generated in the same parent directory as the
+         * parquet file.
+         * <li>When writing multiple parquet files in a single write call, metadata files are only generated if all
+         * parquet files are written to the same parent directory. The metadata files will also be written to the same
+         * parent directory.
```

Comment:
This is inaccurate. If the instructions call for writing metadata files, the code insists that all files be written to the same parent directory.